ompi/mpi/c/profile/Makefile.am

```diff
@@ -0,0 +1,485 @@
+# -*- makefile.am -*-
```

Comment:
Similar to https://github.com/open-mpi/ompi/pull/9707#discussion_r787270200, I think this PR accidentally re-added `ompi/mpi/c/profile/Makefile.am`.